apps/web-giddh/src/app/gst/gstR3/gstR3.component.ts

```diff
@@ -214,7 +214,9 @@ export class FileGstR3Component implements OnInit, OnDestroy {
         if (!this.userEmail) {
             return this._toasty.errorToast("Email Id can't be empty");
         }
-        let monthToSend = moment(this.selectedMonth).format("MM") + "-" + moment(this.selectedMonth).format("YYYY");
+        // Note:- appended ",1" with selectedMonth (July 2020) because "July 2020" format does not support for firefox browser and ("July 2020, 1") is valid format for chrome and firefox browser  
+        let convertValidDateFormatForMoment = this.selectedMonth + ',1';
+        let monthToSend = moment(convertValidDateFormatForMoment).format("MM") + "-" + moment(convertValidDateFormatForMoment).format("YYYY");
```

Comment:
Consider using
`moment(dateRange.fromDate, GIDDH_DATE_FORMAT_MM_DD_YYYY).format(GIDDH_NEW_DATE_FORMAT_UI)` where `GIDDH_DATE_FORMAT_MM_DD_YYYY` is the format in which the date is currently in and `GIDDH_NEW_DATE_FORMAT_UI` is the format in which you want to convert.

Reply:
This is not requirement we need conversion like "July 2020" into "01-2020"